.github/scripts/end2end/configure-e2e-pra.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -exu
+
+. "$(dirname $0)/common.sh"
+
+OIDC_USERNAME=${OIDC_USERNAME:-zenko-end2end-pra}
+OIDC_EMAIL=${OIDC_EMAIL:-e2e-pra@zenko.local}
```

Comment:
these are not used

Reply:
They are used in common.sh for get_token